config/environments/test.rb

```diff
@@ -37,6 +37,9 @@
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
+  # Action Mailer default settings
+  config.action_mailer.default_options = { from: 'noreply@example.org' }
```

Comment:
It is only used in test environment. If I change this into `ENV` the test will fail.